src/Services/Capsules/HasCapsules.php

```diff
@@ -65,6 +80,12 @@ public function getCapsulesSubdir()
 
     public function makeCapsule($capsule, $basePath = null): array
     {
+        $capsule['composer'] = $capsule['composer'] ?? false;
+
+        if ($capsule['composer'] === true) {
+            $basePath =  __DIR__ . '/../../../vendor/' . $capsule['fullName'];
```

Comment:
```suggestion
            $basePath = InstalledVersions::getInstallPath($capsule['fullName']);
```
If I'm not mistaken, this will be required to support Composer `'type' => 'path'` installations, that can be used in development setups. For example, my own development copy of Twill is installed via git submodule into a Laravel project, and then symlinked into `vendor/area17/twill`. The path used above fails to locate the actual install location of the `vendor` directory in this context.

Reply:
:+1: I wasn't aware of that method. That is much better, IMO lots of `../` is a code smell